bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.net;
+
+import java.net.Inet6Address;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.Enumeration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Some utility functions related to network interfaces etc.
+ *
+ * @author Markus Rathgeb - Initial contribution and API
+ */
+public class NetUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NetUtil.class);
```

Comment:
Yes, definitely.
This method has been introduced by Kai.
I moved the method an utility class and make this utility function static.
So, how to use a non-static logger in a static context?

We agreed that in static context we can use a static logger.